docs/HowTo/Get-Started/Use-External-Signer.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Use Eth2Signer for signing
+---
+
+# Use an external signer
+
+Teku supports the [Eth2Signer] external signing client.
+
+**Prerequisites**:
+
+* Eth2Signer installed and running
+* Configured signing key configuration files
+
+## Start Teku
+
+Start Teku and specify the external signer options.
+
+```bash
+teku --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+--eth1-endpoint=http://localhost:8545 --validators-key-file=validator_keys \
```

Comment:
I think using both the validators keys from the key file and using an external signer together is an uncommon scenario so would probably remove the --validators-key-file=validator_keys from the example

Reply:
Done